ros/api/v0/recommendations.py

```diff
@@ -53,7 +53,34 @@ def get(self, host_id):
                 if rule_data:
                     rule_dict = rule_data.__dict__
                     recommendation = {}
+                    instance_price = 0
+                    candidates = rule_hit.get('details').get('candidates')
+                    summaries = rule_hit.get('details').get('summary')
+                    instance_price += rule_hit.get('details').get('price')
```

Comment:
```suggestion
                    instance_price = f'{float(rule_hit_details.get('price'))} USD/hour'
```
As you are formatting `candidates` pricing with unit inside this action, will it OK to construct instance price value with unit `USD/hour` ? any thoughts?

Reply:
@r14chandra -  Based on your changes related to instance price,  please update rules content in seed file.

Reply:
@kgaikwad I'm initializing `instance_price`  because Flake8 is giving error- unused variable. Hence, Build is failing.